components/mpu60xx/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+idf_component_register(SRCS "mpu60xx.c"
+                    INCLUDE_DIRS "include")
```

Comment:
My initial idea was to release this version as v2.0.0 of the original mpu6050 component. 

This way we won't have 2 driver for the same device...